src/content/api/contextmodulefactory-hooks.md

```diff
@@ -0,0 +1,55 @@
+---
+title: ContextModuleFactory Hooks
+group: Plugins
+sort: 11
+contributors:
+  - iguessitsokay
+---
+
+The `ContextModuleFactory` module is used by the `Compiler` to generate dependencies from webpack specific [require.context](/api/module-methods/#requirecontext) API. It resolves the requested directory, generates requests for each file and filters against passed regExp. Matching dependencies then passes through [NormalModuleFactory](/api/normalmodulefactory-hooks).
+
+The `ContextModuleFactory` class extends `Tapable` and provides the following
+lifecycle hooks. They can be tapped the same way as compiler hooks:
+
+``` js
+ContextModuleFactory.hooks.someHook.tap(/* ... */);
+```
+
+As with the `compiler`, `tapAsync` and `tapPromise` may also be available
+depending on the type of hook.
+
+
+### `beforeResolve`
+
+`AsyncSeriesWaterfallHook`
+
+Fired before resolving the requested directory. The request can be ignored by returning `false`.
```

Comment:
```suggestion
Called before resolving the requested directory. The request can be ignored by returning `false`.
```
`Fired` is good in my opinion, unfortunately the linter doesn't think so, thus let's use `called` here.